### DISCUSSION TITLE
V2.17 and Managed Package

### DISCUSSION THREAD

**Discussion by wallygitgud** [SFDO-Community/declarative-lookup-rollup-summaries#1322] [Q&A]
Hello
I was wondering why the managed package version in our Production still v2.11 but the current release is 2.17?

Do I need to update it or will this be updated as per the managed package?

Thank you

**Comment by aheber** [comment]
@wallygitgud DLRS does not push updates (Salesforce allows it but this project doesn't do it), you'll need to run the update on your own using the instructions found here, https://sfdo-community-sprints.github.io/DLRS-Documentation/Installation/
